client/pages/product/updateproduct.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { axiosInstance } from '../../configs/axios';
+
+const UpdateProduct = ({ product }) => {
+  const [updatedProduct, setUpdatedProduct] = useState({
```

Comment:
This is wrong, you should not update the state from props, plus why is this required in this pattern?